libraries/lib-datahandler/src/main/thrift/licenseinfo.thrift

```diff
@@ -70,6 +70,10 @@ struct LicenseInfoParsingResult {
     31: optional string name,
     32: optional string version,
     33: optional string componentType,
+
+    42: optional set<string> componentLanguages,
```

Comment:
maybe the point here, agreeing with @maierthomas is that we have this not as part of the backend service API for the license info service - it appears rather as information that is processed internally to generate the document. Thus I also propose to pull this off from the API of the license inform service.